s2i/bin/assemble

```diff
@@ -17,6 +17,11 @@ echo "---> Running the httpd-24 assemble script..."
 /usr/libexec/s2i/assemble-httpd
 echo "---> Done running the httpd-24 assemble script"
 
+if [ "$(ls /tmp/artifacts/ 2>/dev/null)" ]; then
+  echo "---> Restoring build artifacts (node_modules)"
+  mv /tmp/artifacts/. ./
```

Comment:
I change this line to `mv /tmp/artifacts/node_modules ./` and it seemed to work locally.

Reply:
Not sure how I ended up making the line like this... Trying it with the correct path

Reply:
Even with the line corrected, my tests with OpenShift continue to hang on pulling the current image in order to extract artifacts. It may be something to do with the OpenShift configuration I'm using. It looks like incremental builds are not going to be a priority for the project I'm working on after all, so I'm going to drop this for now. I'll leave it up to you whether to close the pull request.